lang/ja.yml

```
@@ -83,110 +83,110 @@ INSPECTOR_TOGGLED: "検査官モードが{有効|無効}になりました。"
-LOOKUP_LOGIN: "{0} ログに記録されます {in|out}。"
-LOOKUP_PAGE: "ページ {0}"
-LOOKUP_PROJECTILE: "{0} {投げた|ショット} {1} {2}。"
-LOOKUP_ROWS_FOUND: "{0} {行|行} 見つかりました。"
-LOOKUP_SEARCHING: "ルックアップ検索。お待ちください..."
-LOOKUP_STORAGE: "{0} {預け入れ|撤回} {1} {2}。"
-LOOKUP_TIME: "{0} 前に"
-LOOKUP_USERNAME: "{1}が{0}としてログイン"
-MAXIMUM_RADIUS: "最大 {ルックアップ|ロールバック|復元} 半径は {0}。"
+LOOKUP_BLOCK: "{0}が{1}を{配置|破壊}した。"
+LOOKUP_CONTAINER: "{0}が{2} {1}を{追加|取り出}した。"
+LOOKUP_HEADER: "{0} の結果"
+LOOKUP_INTERACTION: "{0}が{1}を{クリック|破壊}した。"
+LOOKUP_ITEM: "{0}が{2} {1}を{拾った|落とした}。"
+LOOKUP_LOGIN: "{0}の{ログイン|ログアウト}がログに記録されます。"
+LOOKUP_PAGE: "{0} ページ目"
+LOOKUP_PROJECTILE: "{0}が{2} {1}を{投げた|撃った}。"
+LOOKUP_ROWS_FOUND: "{0}行見つかりました。"
+LOOKUP_SEARCHING: "検索中。しばらくお待ちください..."
+LOOKUP_STORAGE: "{0}が{2} {1}を{預け入れ|引き出し}た。"
+LOOKUP_TIME: "{0}前"
+LOOKUP_USERNAME: "{0}が{1}としてログイン"
+MAXIMUM_RADIUS: "最大半径の{0}中で{検索|ロールバック|復元}をします。"
 MISSING_ACTION_USER: "そのアクションを使用するには、ユーザーを指定してください。"
-MISSING_LOOKUP_TIME: "時間を指定してください({ルックアップ|ロールバック|復元})"
+MISSING_LOOKUP_TIME: "時間を指定してください({検索|ロールバック|復元})"
 MISSING_LOOKUP_USER: "ユーザーを指定するか、 {ブロック|半径}を指定してください。"
 MISSING_PARAMETERS: "\"{0}\"のパラメーターは存在していません。"
-MISSING_ROLLBACK_RADIUS: "指定しませんでした {ロールバック|復元} 半径。"
-MISSING_ROLLBACK_USER: "指定しませんでした {ロールバック|復元} ユーザー。"
-MYSQL_UNAVAILABLE: "接続できません MySQL サーバ。"
+MISSING_ROLLBACK_RADIUS: "{ロールバック|復元}対象の半径を指定しませんでした。"
+MISSING_ROLLBACK_USER: "{ロールバック|復元}対象のユーザーを指定しませんでした。"
+MYSQL_UNAVAILABLE: "MySQLサーバーに接続できません"
 NO_DATA: "{0}でデータが見つかりません。"
-NO_DATA_LOCATION: "番号 {データ|トランザクション|相互作用|メッセージ} 見つかった。"
+NO_DATA_LOCATION: "{データ|トランザクション|相互作用|メッセージ}は見つかりませんでした。"
 NO_PERMISSION: "あなたにはそれをする許可がありません。"
-NO_RESULTS: "結果が見つかりません。"
-NO_RESULTS_PAGE: "番号 {結果|データ} そのページで見つかりました。"
-NO_ROLLBACK: "番号 {保留中|前へ} ロールバック/復元が見つかりました。"
+NO_RESULTS: "結果が見つかりませんでした。"
+NO_RESULTS_PAGE: "このページには{結果|データ}は見つかりませんでした。"
+NO_ROLLBACK: "{保留中の|進んでいる}ロールバック/復元は見つかりませんでした。"
 PATCH_INTERRUPTED: "アップグレードが中断されました。再起動時に再試行します。"
-PATCH_OUTDATED_1: "古いデータベースをアップグレードできません {0}。"
-PATCH_OUTDATED_2: "CoreProtectがサポートされているバージョンでアップグレードしてください。"
-PATCH_PROCESSING: "新しいデータを処理しています。お待ちください..."
-PATCH_SKIP_UPDATE: "スキップ {テーブル|インデックス} {更新|作成|削除} の上 {0}。"
-PATCH_STARTED: "実行する {0} アップグレード。お待ちください..."
-PATCH_SUCCESS: "正常にアップグレードされました {0}。"
-PATCH_UPGRADING: "データベースのアップグレードが進行中です。お待ちください..."
-PLEASE_SELECT: "選んでください: \"{0}\" また \"{1}\" 。"
+PATCH_OUTDATED_1: "{0}は古いデータベースをアップグレードできません。"
+PATCH_OUTDATED_2: "CoreProtect がサポートされているバージョンにアップグレードしてください。"
+PATCH_PROCESSING: "新しいデータを処理しています。しばらくお待ちください..."
+PATCH_SKIP_UPDATE: "{0}の{テーブル|インデックス}の{更新|作成|削除}をスキップします。"
+PATCH_STARTED: "{0} のアップグレードを実行します。しばらくお待ちください..."
+PATCH_SUCCESS: "{0} は正常にアップグレードされました。"
+PATCH_UPGRADING: "データベースのアップグレードが進行中です。しばらくお待ちください..."
+PLEASE_SELECT: "選んでください: \"{0}\" または \"{1}\" 。"
 PREVIEW_CANCELLED: "プレビューがキャンセルされました。"
 PREVIEW_CANCELLING: "プレビューをキャンセルしています..."
 PREVIEW_IN_GAME: "ゲーム内でのみロールバックをプレビューできます。"
-PREVIEW_TRANSACTION: "プレビューできません {コンテナ|在庫} トランザクション。"
+PREVIEW_TRANSACTION: "{コンテナ|在庫}の入れ替えがプレビューできません"
```

```suggestion
PREVIEW_TRANSACTION: "{コンテナ|在庫}}のトランザクションをプレビューすることはできません"
```
ごめん、見逃した。
Sorry, I missed it.